Automattic/syntaxhighlighter
# Fix content attribute selector

Follow-up to #163 

### Changes proposed in this Pull Request

* Change back the selector for content attribute, since the  extra `<code>` wrapper from #160 was not needed after all.
  * This also caused an issue of the block being re-created from the deprecated helper on every open
* This also removes the need for the `deprecated` helper — this would make sure block output from older versions are compatible, but the block output is not going to change in `3.5.6`

### Testing instructions

* Create a code block in `master`
* Switch to this branch
* Edit the post and ensure the block works without any warnings
* Change/add new code blocks, save post.
* Reload the block editor and check that block still works without warning.
